### DISCUSSION TITLE
Can we add ShareX stable version ?

### DISCUSSION THREAD

**Discussion by myagizmaktav** [lutris/lutris#5403] [General]
Hello, can somebody added to sharex to stable version ? 
Thank you.

**Comment by danieljohnson2** [comment]
Is not that a Windows program?

**Reply by myagizmaktav** [reply]
Yes it is.
But Its very useful screenshot.
I start using Linux Ubuntu about 3 month and I still cannot find near alternative like sharex.
I try to made lutris script and made it but it not work pretty well.

**Reply by strycore** [reply]
That's not going to work on Wayland :roll_eyes:

**Reply by myagizmaktav** [reply]
@strycore  what you use for screenshoot?

**Reply by strycore** [reply]
I use Spectacle the default tool in KDE. I've also used Flameshot on different desktops. 

It wouldn't cross my mind to use a Windows tool to do something basic like taking screenshots.
